[Feature] #27 - 초대코드 입력 뷰 및 프로필 입력 뷰 구현

kj9470 opened the pull request:
## 💭 작업 배경
<!-- 아래 리스트를 지우고, 작업하게 된 배경을 적어주세요. -->
- 입력되었을 때 구냥 넘어갈 수 있도록 작업했읍니당

## 🌤️ PR POINT
<!-- 작업 내용 및 덧붙이고 싶은 내용이 있다면! -->

## 📸 스크린샷
<!-- 작업한 화면이 있다면 스크린 샷으로 첨부해주세요. -->

|    구현 내용    |   스크린샷   |
| :-------------: | :----------: |
| GIF | <img src = "https://github.com/Team-Puzzling/Puzzling_iOS/assets/48792069/3905633b-75e6-412f-ae80-2869319ccf9d" width ="250">|

## 🌈 관련 이슈
<!-- 작업한 이슈번호를 # 뒤에 붙여주세요. 수고했습니다~* -->
- Resolved: #27

kj9470 committed:
[Add] #27 - 이미지 에셋 추가

kj9470 committed:
[Design] #27 - 초대코드 입력 뷰 UI 구현

kj9470 committed:
[Feature] #27 - 키보드 처리

kj9470 committed:
[Feature] #27 - 바탕화면 눌렀을 시 키보드 종료

kj9470 committed:
[Chore] #27 - 텍스트필드 경고 메시지 분기처리

kj9470 committed:
[Feature] #27 - 텍스트 있을 시 버튼 활성화

kj9470 committed:
[Feature] #27 - 프로필 작성 뷰컨 구현

frohsch reviewed (approved):
수고하셨습니다 !!

Hong0329 reviewed (approved):
권! 정! 홍준!

amourxyoung commented on Puzzling/Puzzling/Network/Foundation/URLConst.swift:
```diff
@@ -11,17 +11,36 @@ public enum URLConst {
+    static let postReviewAAR = "/review/ARR"
+    static let getReviewPreviousTemplateURL = "/review/previous-template"
+    
+    static let postJoinURL = "/project/join"
```
우와 이런식으로 url 상수 처리할수도 있군요 ! 배워갑니당

kj9470 replied:
우히히 우하하 우하하 웅히히

yeseul106 reviewed (approved):
역시 아요 리드는 다르시군요 정말 멋진 코드 감탄하고 갑니다.

amourxyoung reviewed (approved):
역시 리드님 짱이시네요 👍 코드 잘보고 갑니당

kj9470 committed:
[Feature] #27 - 화면전환 구현

kj9470 committed:
[Merge] #27 - Commit Merge

Merge branch 'develop' of https://github.com/Team-Puzzling/Puzzling_iOS into feature/#27
